modules/hal_nordic/nrfx/nrfx_config_nrf54h20_enga_ppr.h

```diff
@@ -0,0 +1,1877 @@
+/*
+ * Copyright (c) 2023, Nordic Semiconductor ASA
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
```

Comment:
Shorter copyright notice, like in the other `modules/hal_nordic/nrfx/nrfx_config_*.h` files?

Reply:
adjusted to apache, copy paste artifact from hal templates